docs/jobs/openings/web3.md

```diff
@@ -0,0 +1,34 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us consolidate smart contracts with a frontend, we call this position The Glue. Our Web 3 Developer must not only be a Javascript rockstar but also a decentralization evangelist, someone who enjoys researching and testing new Web3 technologies.
```

Comment:
I'd rephrase to:

> We’re looking for a Web3 Developer to help us **bridge** smart contracts **to** a frontend. **Internally, we call this position "Glue".** Our Web3 Developer **is ideally** not only a Javascript expert but also a decentralization evangelist—someone who enjoys researching and testing new Web3 technologies.

Reply:
To be honest, "a frontend" also sounds super vague, either do "frontends" or "the frontend" (I'd suggest frontends since we're talking about multiple representations, not just aragon/aragon)